doglist.html

```diff
@@ -0,0 +1,119 @@
+  <p>Adopt</p>
+
+  <img src="./images/duffy-card.jpg" alt="picture of dog named duffy">
+  <h3>Duffy</h3>
+  <p>Cost to Adopt: $123.45</p>
+  <p>Corrum volorit iandae nimaxim cum restia volor reicid ut et etur sunt arum rendae pla endis re ea erum, qui doluptae</p>
+  <p>Adopt</p>
+
+  <img src="./images/lucas-card.jpg" alt="picture of dog named lucas">  <h3>Lucas</h3>
+  <p>Cost to Adopt: $123.45</p>
+  <p>Corrum volorit iandae nimaxim cum restia volor reicid ut et etur sunt arum rendae pla endis re ea erum, qui doluptae</p>
+  <p>Adopt</p>
+
+  <img src="./images/jake-card.jpg" alt="picture of dog named jake">
+  <h3>Jake</h3>
+  <p>Cost to Adopt: $123.45</p>
+  <p>Corrum volorit iandae nimaxim cum restia volor reicid ut et etur sunt arum rendae pla endis re ea erum, qui doluptae</p>
+  <p>Adopt</p>
+
+  <img src="./images/angus-card.jpg" alt="picture of dog named angus">
+  <h3>Angus</h3>
+  <p>Cost to Adopt: $123.45</p>
+  <p>Corrum volorit iandae nimaxim cum restia volor reicid ut et etur sunt arum rendae pla endis re ea erum, qui doluptae</p>
+  <p>Adopt</p>
+
+  <img src="./images/violet-card.jpg" alt="picture of dog named violet">
+  <h3>Violet</h3>
+  <p>Cost to Adopt: $123.45</p>
+  <p>Corrum volorit iandae nimaxim cum restia volor reicid ut et etur sunt arum rendae pla endis re ea erum, qui doluptae</p>
+  <p>Adopt</p>
+
+  <img src="./images/piper-card.jpg" alt="picture of dog named piper">
+  <h3>Piper</h3>
+  <p>Cost to Adopt: $123.45</p>
+  <p>Corrum volorit iandae nimaxim cum restia volor reicid ut et etur sunt arum rendae pla endis re ea erum, qui doluptae</p>
+  <p>Adopt</p>
+
+  <img src="./images/maximus-card.jpg" alt="picture of dog named maximus">
+  <h3>Maximus</h3>
+  <p>Cost to Adopt: $123.45</p>
+  <p>Corrum volorit iandae nimaxim cum restia volor reicid ut et etur sunt arum rendae pla endis re ea erum, qui doluptae</p>
+  <p>Adopt</p>
+
+  <img src="./images/luna-card.jpg" alt="picture of dog named luna">
+  <h3>Luna</h3>
+  <p>Cost to Adopt: $123.45</p>
+  <p>Corrum volorit iandae nimaxim cum restia volor reicid ut et etur sunt arum rendae pla endis re ea erum, qui doluptae</p>
+  <p>Adopt</p>
+
+  <img src="./images/stella-card.jpg" alt="picture of dog named stella">
+  <h3>Stella</h3>
+  <p>Cost to Adopt: $123.45</p>
+  <p>Corrum volorit iandae nimaxim cum restia volor reicid ut et etur sunt arum rendae pla endis re ea erum, qui doluptae</p>
+  <p>Adopt</p>
+</section>
+
+
+</main>
+<footer>
+  <img src="./images/adoptadog-logo.svg" alt="adopt a dog logo">
+<p>Aborpos nisto inci tet aut que volupti beruptiberro to ex  esseque invenienihil imaximu scideru ptistiusam consequunt</p>
+
+<nav>
+    <a class="footernavigationitem" href="./index.html">Home</a>
+    <span>  /  </span>
+    <a class="footernavigationitem" href="./doglist.html">Dogs</a>
+    <span>  /  </span>
+    <a class="footernavigationitem">Contact</a>
+</nav>
+  <p>2018 Adoptadog, LLC  |  Seattle, WA</p>
```

Comment:
don't forget the copy right symbol. You can find out how to do this by looking up HTML symbols